### ISSUE TITLE
Possible future feature 

### ISSUE THREAD

**Issue by vsiri** [Up-Next/up-next#86]
- Add option for private or public parties 
- Popular public parties can get featured on the homepage

**Comment by vsiri** [comment]
Requested by a friend bc the site is intimidating for a noob and a popular parties section can get new people into the site

**Comment by kimberlytze** [comment]
How would private parties work? A password to access it?

**Comment by VOjha** [comment]
^ That's probably the best option? Or maybe a list of usernames that are authorized has to be provided? Which would be a lot of work lol...

**Comment by kimberlytze** [comment]
List of usernames sounds like more work to me. Because then someone (the host?) has to go and authorize them all in the first place

**Comment by VOjha** [comment]
Right, which is why I said it would be a lot of work. Just throwing out ideas.

**Comment by vsiri** [comment]
i think passwords sound good 👍
